### PR TITLE
fix(Enumeration): same size on the default or edit mode

### PR DESCRIPTION
**Pull request by jare-talend**
**Please check if the PR fulfills these requirements**

- [x] The commit(s) message(s) follows our
  [guidelines](https://github.com/talend/tools/blob/master/tools-root-github/CONTRIBUTING.md#commit-message-format)
- [ ] Tests for the changes have been added (for bug fixes / features)
- [ ] Docs have been added / updated (for bug fixes / features)

**What is the current behavior?** (You can also link to an open issue here)
The size's items are different

**What is the new behavior?**
The same size for the item in the state EDIT or DEFAULT . need this PR(https://github.com/Talend/ui/pull/275) to change the font-size actually at 16px to 14px

**Does this PR introduce a breaking change?**

- [ ] Yes
- [ ] No

If this PR contains a breaking change, please describe the impact and migration
path for existing applications: ...

**Other information**:

### CODE REVIEW

**Comment by jmfrancois** [comment]
:octocat: [Demo is available here](http://276.talend.surge.sh)

**Comment by jmfrancois** [comment]
:octocat: [Demo is available here](http://276.talend.surge.sh)

File: packages/components/src/Enumeration/Items/Item/Item.scss [hunk 1]
```diff
@@ -55,7 +55,8 @@ $tc-enumeration-small-padding: $padding-small !default;
 	}
 
 	input {
-		margin: 10px;
+		line-height: 3.3rem;
```
Comment (frassinier) [inline packages/components/src/Enumeration/Items/Item/Item.scss]: Why 3.3rem? Where does it come from?

File: packages/components/src/Enumeration/Items/Item/Item.scss [hunk 2]
```diff
@@ -55,7 +55,8 @@ $tc-enumeration-small-padding: $padding-small !default;
...
+		margin: 0px 15px;
```
Comment (frassinier) [inline packages/components/src/Enumeration/Items/Item/Item.scss]: Try using [these values](https://github.com/Talend/ui/blob/master/packages/theme/src/theme/_paddings.scss) please

File: packages/components/src/Enumeration/Items/Item/Item.scss (hunk 1 above)
Comment (jare-talend) [inline packages/components/src/Enumeration/Items/Item/Item.scss]: In the default state, a button wraps a item which define the height of a item with this property. So, I have added this property to get the same behavior.

**Comment by jmfrancois** [comment]
:octocat: [Demo is available here](http://276.talend.surge.sh)

**Comment by jmfrancois** [comment]
:octocat: [Demo is available here](http://276.talend.surge.sh)

**Comment by jmfrancois** [comment]
:octocat: [Demo is available here](http://276.talend.surge.sh)

File: packages/components/src/Enumeration/Items/Item/Item.scss [hunk 3]
```diff
@@ -55,7 +55,8 @@ $tc-enumeration-small-padding: $padding-small !default;
 	}
 
 	input {
-		margin: 10px;
+		height: 3.4rem;
+		margin: 0px $padding-normal;
```
Comment (jmfrancois) [inline packages/components/src/Enumeration/Items/Item/Item.scss]: warning  No unit allowed for values of 0  zero-unit

**Comment by jmfrancois** [comment]
:octocat: [Demo is available here](http://276.talend.surge.sh)

File: packages/components/src/Enumeration/Items/Item/Item.scss [hunk 4]
```diff
@@ -55,7 +55,8 @@ $tc-enumeration-small-padding: $padding-small !default;
 	}
 
 	input {
-		margin: 10px;
+		height: 3.4rem;
```
Comment (frassinier) [inline packages/components/src/Enumeration/Items/Item/Item.scss]: Looks like a magic number

**Comment by jmfrancois** [comment]
:octocat: [Demo is available here](http://276.talend.surge.sh)

**Comment by jmfrancois** [comment]
:octocat: [Demo is available here](http://276.talend.surge.sh)

**Comment by jmfrancois** [comment]
:octocat: [Demo is available here](http://276.talend.surge.sh)

**Comment by jmfrancois** [comment]
:octocat: [Demo is available here](http://276.talend.surge.sh)

**Comment by jmfrancois** [comment]
:octocat: [Demo is available here](http://276.talend.surge.sh)

**Comment by jmfrancois** [comment]
:octocat: [Demo is available here](http://276.talend.surge.sh)

**Comment by jmfrancois** [comment]
:octocat: [Demo is available here](http://276.talend.surge.sh)

**Pull request merged**